lib/interpolateName.js

```diff
@@ -64,12 +64,18 @@ function interpolateName(loaderContext, name, options) {
 		}
 	}
 	let url = filename;
+	const compilationOutputOptions = loaderContext._compilation && loaderContext._compilation.outputOptions || {};
```

Comment:
It is private properties and don't use their, you need open issue in webpack with feature request provide this option throw api

Reply:
@evilebottnawi thanks, i will do it

Reply:
@aarefiev good news, thanks for helping

Reply:
@aarefiev can you create issue so that we don't lose this problem if it require time?

Reply:
@evilebottnawi done (https://github.com/webpack/loader-utils/issues/121)

Reply:
I will send pr for webpack in next few days.